docs/src/inverse_problems/petab_ode_param_fitting.md

```diff
@@ -0,0 +1,487 @@
+```petab1
+using Optim
+p0 = generate_startguesses(petab_problem, 1)
+res = calibrate_model(petab_problem, p0, IPNewton())
+```
+
+We can now simulate our model for the fitted parameter set, and compare the result to the measurements and true solution.
+```petab1
+oprob = ODEProblem(rn, u0, (0.0, 10.0), get_ps(res, petab_problem))
+
+sol = solve(oprob, Tsit5(); saveat=0.1)
+plot!(sol; idxs=4, label="Fitted solution")
+```
+Here we use the `get_ps` function to retrieve a full parameter set using the optimal parameters (alternatively, the `ODEProblem` or fitted simulation can be retrieved directly using the `get_odeproblem` or `get_odesol` [functions](https://sebapersson.github.io/PEtab.jl/dev/API_choosen/#PEtab.get_odeproblem), respectively). The calibration result can also be found in `res.xmin`, however, note that PEtab automatically ([unless a linear scale is selected](petab_parameters_scales)) converts parameters to logarithmic scale, so typically `10 .^res.xmin` are the values of interest. If you investigate the result from this example you might note that the found parameter set (while it fits the data well) does not correspond to the true parameter set. This phenomenon is related to *identifiability*, and is very important for parameter fitting.
+
+### Final notes
+PEtab.jl also supports [multistart optimisation](@ref petab_multistart_optimisation), [automatic equilibration before simulations](https://sebapersson.github.io/PEtab.jl/stable/Brannmark/), and [events](@ref petab_events). Various [plot recipes](@ref petab_plotting) exist for investigating the optimisation process. Please read the [PETab.jl documentation](https://sebapersson.github.io/PEtab.jl/stable/) for a more complete description of the package's features. Below follows additional details of various options and features (generally, PEtab is able to find good default values for most options not specified).
+
+## [Additional features: Observables](@id petab_observables)
+
+### [Defining non-trivial observables](@id petab_observables_observables)
+It is possible for observables to be any algebraic expression of species concentrations and parameters. E.g. in this example the total amount of `X` in the system is an observable:
+```petab2
+using Catalyst, PEtab # hide
+rn = @reaction_network begin
+    (k1,k2), X1 <--> X2
+end
+@unpack X1, X2 = rn
+obs_X = PEtabObservable(X1 + X2, 0.5)
+```
+
+### [Advanced observables noise formulas](@id petab_observables_noise_formula)
+In our basic example we assumed that the normally distributed noise had a standard deviation of `0.5`. However, this value may be a parameter (or indeed any algebraic expression). E.g, we could set
+```petab1
+@parameters σ
+obs_P = PEtabObservable(P, σ)
+```
+and then let the parameter *σ* vary between different [simulation conditions](@ref petab_simulation_conditions). Alternatively we could let the noise scale linearly with the species' concentration:
```

Comment:
would also add, or we could estimate *\sigma* (add it as a PEtabParameter)

Estimating noise parameters is quite popular

Reply:
added this